Application/Favorites/CreateFavorite/CreateFavoriteCommandHandler.cs

```diff
@@ -0,0 +1,57 @@
+using Application.DataAccess;
+using Domain.Favorites;
+using Domain.Favorites.Exceptions;
+using Domain.Tracks;
+using Domain.Tracks.Exceptions;
+using Domain.Users;
+using Domain.Users.Exceptions;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace Application.Favorites.CreateFavorite;
+
+public class CreateFavoriteCommandHandler: IRequestHandler<CreateFavoriteCommand, Result<bool>>
+{
+    private readonly IApplicationDbContext _context;
+
+    public CreateFavoriteCommandHandler(IApplicationDbContext context)
+    {
+        _context = context;
+    }
+    
+    public async Task<Result<bool>> Handle(CreateFavoriteCommand request, CancellationToken cancellationToken)
+    {
+        User? user = _context.Users.FirstOrDefaultAsync(u => u.Id == request.UserId).Result;
```

Comment:
WTF? Это что за пьяное программирование? Везде на вот так надо поменять.
```suggestion
        User? user = await _context.Users.FirstOrDefaultAsync(u => u.Id == request.UserId);
```

Reply:
На улице апрель, а новый год будто бы вчера